src/cachers/entities/organisations/set.js

```diff
@@ -0,0 +1,84 @@
+const utils = require('./utils');
+
+let axiosClient;
+let redisClient;
+
+const pageSize = 100;
+
+const pageOfOrganisationResults = page => {
+  return axiosClient.get('/search', {
+    params: {
+      ...axiosClient.defaults.params,
+      query: '*:*',
+      type: 'organization',
+      page,
+      pageSize
+    }
+  })
+    .then(response => response.data.items)
+    .then(items => items ? items.map(persistableFields) : []);
+};
+
+const allOrganisationResults = async() => {
+  let allResults = [];
+  let page = 0; // Yes, the Entity API pagination starts at page 0. ¯\_(ツ)_/¯
+  let pageOfResults;
+
+  // the API allows 100 entities per request. Loop until all organisations are retrieved.
+  while (!Array.isArray(pageOfResults) || pageOfResults.length > 0) {
+    pageOfResults = await pageOfOrganisationResults(page);
+    allResults = allResults.concat(pageOfResults);
+    page = page + 1;
+  }
+
+  return allResults;
+};
+
+const organisationsObject = results => results
+  .reduce((memo, { identifier, prefLabel }) => {
+    memo[identifier[0]] = { prefLabel };
+    return memo;
+  }, {});
+
+const persistableFields = ({ identifier, prefLabel }) => {
+  return {
+    identifier,
+    prefLabel
+  };
+};
+
+const writeToRedis = (organisations) => {
+  return redisClient.setAsync(utils.CACHE_KEY, JSON.stringify(organisations, null, 2))
```

Comment:
the extra spacing passed as an option to stringify here results in considerable increase in the amount of data stored.
Without it, the whole organisation data string is 393.46kB while with it I get 516.5kB
It shouldn't really be necessary to read what's in this redis cache manually.